apps/admin-gui/src/app/vos/pages/group-detail-page/group-detail-page.component.html

```diff
@@ -43,16 +43,12 @@
             queryParamsHandling="merge"
             >{{vo.name}}</a
           >, {{'GROUP_DETAIL.DESCRIPTION' | translate}}: {{group.description}}
-          <span *ngIf="syncEnabled">
+          <span>
             , {{'GROUP_DETAIL.SYNCHRONIZATION' | translate}}:
             <span
               [matTooltipPosition]="'above'"
               [matTooltip]="group | groupSyncToolTip | translate">
-              <button
-                (click)="onSyncDetail()"
-                *ngIf="syncAuth"
-                [disabled]="(group | groupSyncIcon) === 'sync_disabled'"
-                mat-icon-button>
+              <button (click)="onSyncDetail()" mat-icon-button>
```

Comment:
Here we lost the information if user is authorized to force group synchornization. We can move this check to the dialog itself so we can still open it but cannot perform the action.